diff --git a/index.d.ts b/index.d.ts
index 2c5f3c56393577d8c245ae7ba03e33ebda943a60..4f8bd887c68d6a8577b452f142641f6ab36e95f3 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -2,6 +2,8 @@
  * The JavaScript API for embedding Apple maps on your website.
  */
 declare namespace mapkit {
+    const loadedLibraries: readonly string[];
+
     /**
      * Initialize a mapkit object by providing an authorization callback and language.
      */
@@ -1862,7 +1864,7 @@ declare namespace mapkit {
             query: string,
             callback: SearchDelegate | AutocompleteSearchCallback,
             options?: SearchAutocompleteOptions,
-        ): void;
+        ): number;
         /**
          * Cancels a search request using its request ID.
          *
